src/mixins/lit-localize-lite-mixin.ts

```diff
@@ -19,23 +19,29 @@ export const litLocalizeLiteMixin = <T extends Constructor<LitElement>>(
     public connectedCallback(): void {
       super.connectedCallback();
       this._initializeLocalizeLite();
-      this.localize = computeLocalize(
-        this.constructor.prototype,
-        this.language!,
-        this.resources!
-      );
     }
 
     protected updated(changedProperties: PropertyValues) {
       super.updated(changedProperties);
+      if (changedProperties.get("translationFragment")) {
+        this._initializeLocalizeLite();
+      }
+      if (changedProperties.has("language")) {
+        const html = document.querySelector("html")!;
+        if (html.getAttribute("lang") !== this.language) {
+          html.setAttribute("lang", this.language!);
+        }
+      }
```

Comment:
Even if it's only used at a couple of pages, let's still not put it here. Let's put it in the main elements for onboarding/authorize.